packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -151,6 +150,44 @@ export interface StatefulCallClient extends CallClient {
     stream: LocalVideoStreamState | RemoteVideoStreamState
   ): void;
 
+  /* @conditional-compile-remove(together-mode) */
+  /**
+   * Renders a {@link CallFeatureStreamState}
+   * {@link VideoStreamRendererViewState} under the relevant {@link CallFeatureStreamState}
+   * {@link @azure/communication-calling#VideoStreamRenderer.createView}.
+   *
+   * Scenario 1: Render CallFeatureStreamState
+   * - CallId is required and stream of type CallFeatureStreamState is required
+   * - Resulting {@link VideoStreamRendererViewState} is stored in the given callId and participantId in
+   * {@link CallClientState}
+   *
+   * @param callId - CallId for the given stream. Can be undefined if the stream is not part of any call.
+   * @param stream - The LocalVideoStreamState or RemoteVideoStreamState to start rendering.
+   * @param options - Options that are passed to the {@link @azure/communication-calling#VideoStreamRenderer}.
+   * @beta
+   */
+  createCallFeatureView(
```

Comment:
I wasn't around for the discussions on this, can you talk about why this is a whole new function? I see there is also a new streamUtil function - but looks very similar to the existing createView function, what is the differences between createCallFeatureView and createView?

Reply:
Current implementation of creating stream views is either tied to a participant in the call (parented)  or to the device manager (unparented). It currently does not support situations where there's a bot stream because bot IDs are hidden from the call roster.  Although there are similarities, between stream utils file and the call feature stream utils, it was advised to separate them for now to avoid introducing blocking bugs

Reply:
Just want to ack on this but haven't had a chance to relook yet

Reply:
Splitting into two issues. Will make this thread just about public API.
Can we make this an overload? i.e.
```ts
createView(
  callId: string,
  stream: CallFeatureStreamState,
  options?: CreateViewOptions
);
```

Reply:
Updated to overloaded method